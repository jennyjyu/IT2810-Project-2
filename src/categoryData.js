const categoryData = [
    {
        id: 1,
        mediaLabel: "Images",
        categories :[
            {
                id: 1,
                text: "Animals",
                name: "animals",
                checked: true,

            },
            {
                id: 2,
                text: "Vehicles",
                name: "vehicles",
                checked: false,
            },
            {
                id: 3,
                text: "Nature",
                name: "nature",
                checked: false,
            }
        ]
    },
    {
        id: 2,
        mediaLabel: "Sounds",
        categories : [
            {
                id: 1,
                text: "Cartoons",
                name: "cartoons",
                checked: true
            },
            {
                id: 2,
                text: "Jazz",
                name: "jazz",
                checked: false
            },
            {
                id: 3,
                text: "Horror",
                name: "horror",
                checked: false
            }

        ]
    },
    {
        id: 3,
        mediaLabel: "Texts",
        categories : [
            {
                id: 1,
                text: "Animals",
                name: "animals",
                checked: true
            },
            {
                id: 2,
                text: "Horror",
                name: "horror",
                checked: false
            },
            {
                id: 3,
                text: "Jokes",
                name: "jokes",
                checked: false
            }

        ]
    }
]

export default categoryData
