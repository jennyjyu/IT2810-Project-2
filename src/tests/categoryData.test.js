import React from "react"
import categoryData from "../categoryData"
import renderer from "react-test-renderer"



test("categoryData snapshot test", () => {
  const component = renderer.create(
    <categoryData />
  )
  let tree = component.toJSON()

  expect(tree).toMatchSnapshot()
})
