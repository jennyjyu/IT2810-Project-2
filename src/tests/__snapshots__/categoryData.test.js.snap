// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`categoryData snapshot test 1`] = `<categoryData />`;
